### PR TITLE
Use clang-format

### LINKED ISSUE #349: Move from astyle to clang-format

### PR DESCRIPTION
**Pull request by patrickelectric**
- Use clang-format over astyle
- Move style changes from astyle to clang-format 
- Move ping-viewer source code from our own style to a more generic style compatible with Qt and with some minor changes

Fix #349

### CODE REVIEW

File: src/util/util.cpp [hunk 1]
```diff
@@ -33,38 +31,38 @@ void Util::update(QtCharts::QAbstractSeries* series, const QVector<double>& poin
...
     }
 
     // Data
-    const int lastDataPoint = int((finalPos - initPos)*distPoints);
-    const float dataIndexScale = points.length()/((finalPos - initPos)*distPoints);
-    #pragma omp for
-    for(int i = 0; i < lastDataPoint; i++) {
-        realPoints << QPointF(i + lastStartPoint, multiplier * points[static_cast<int>(i*dataIndexScale)]);
+    const int lastDataPoint = int((finalPos - initPos) * distPoints);
+    const float dataIndexScale = points.length() / ((finalPos - initPos) * distPoints);
```
Comment (jaxxzer) [inline src/util/util.cpp]: want to change the math rules?

**Comment by jaxxzer** [comment]
looking good

File: src/util/util.cpp (hunk 1 above)
Comment (patrickelectric) [reply src/util/util.cpp]: I was unable to find any math format in clang for that, I believe that will need to drop that if we are aiming to use clang-format.
@tcanabrava, any tips ?

**Comment by Williangalvani** [comment]
I like it. Can you update Travis to check the style with it?

**Comment by patrickelectric** [comment]
@Williangalvani check :
- clang-format: Replace astyle script … de09185

**Comment by Williangalvani** [comment]
> @Williangalvani check :
> 
> * clang-format: Replace astyle script … [de09185](https://github.com/bluerobotics/ping-viewer/commit/de09185268731c146f229957f5dcd07074ed9321)

Right, I saw travis was failing during style check and thought you hadn't moved to clang-format there, my bad.
Can you fix it?

**Comment by patrickelectric** [comment]
@Williangalvani Yes, I'm still checking travis configuration since I'm unable to run it locally.

**Comment by patrickelectric** [comment]
Travis fixed

File: .travis.yml [hunk 2]
```diff
@@ -16,8 +16,10 @@ matrix:
             - sourceline: 'ppa:cs50/ppa'
             - sourceline: 'ppa:ubuntu-toolchain-r/test'
             - sourceline: 'ppa:beineri/opt-qt-5.12.2-xenial'
+            - sourceline: 'deb http://apt.llvm.org/stretch/ llvm-toolchain-stretch-9 main'
+              key_url: 'https://apt.llvm.org/llvm-snapshot.gpg.key'
           packages:
-            - astyle
+            - clang-format-9
```
Comment (jaxxzer) [inline .travis.yml]: Why do we need to uninstall and use clang-9?

It doesn't look like you are using any new formatting options.

Comment (patrickelectric) [reply .travis.yml]: No new options, but clang-format 7 does have some bugs for this features.

**Pull request merged**